setup.py

```diff
@@ -29,15 +28,16 @@ def read_requirements(path: str) -> List[str]:
     return ret
 
 
-def read(*names: str, **kwargs: str) -> str:
-    return io.open(join(dirname(__file__), *names), encoding=kwargs.get("encoding", "utf8")).read()
+with open("README.md") as readme_file:
+    README = readme_file.read()
 
 
 setup(
     name="raiden-services",
     version="0.4.0",
     license="MIT",
     description=DESCRIPTION,
+    long_description=README,
```

Comment:
Do we need to add
```
long_description_content_type='text/markdown'
```
?

Reply:
Good point, I missed that in the raiden setup config.